index.html

```diff
@@ -654,5 +654,368 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
     </p>
     </div>
   </section>
+  <section>
+    <h2>Human face segmentation</h2>
+    <p>Human face metadata describes the geometrical information of human faces in video frames. It can
+    be set by web applications using the standard means when creating
+    {{VideoFrameMetadata}} for {{VideoFrame}}s or it can be set by a user agent
+    when the media track constraints, defined below, are used to
+    enable face or face landmark detection for the {{MediaStreamTrack}} which provides the
+    {{VideoFrame}}s.</p>
+    <p>The facial metadata can be used by video encoders to enhance the quality
+    of the faces in encoded video streams or for other suitable purposes.</p>
+    <section>
+      <h3>{{VideoFrameMetadata}}</h3>
+      <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  sequence&lt;Segment&gt; segment;
+};</pre>
+      <section class="notoc">
+        <h4>Members</h4>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>segment</code></dfn> of type <code>sequence&lt;{{Segment}}&gt;</code></dt>
+          <dd>
+            <p>The set of known geometrical segments in the video frame.</p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section>
+      <h3>{{Segment}}</h3>
+      <pre class="idl">
+dictionary Segment {
+  required SegmentType type;
+  required long        id;
```

Comment:
The requirement is probably for the segments sequence to not contain two segment dictionaries with the same id.
Somehow, it seems the segments should be map-like (id -> Segment) instead of sequence.

Reply:
Yes, it is the requirement. But it looks that maplike is for interfaces, not dictionaries. Changing to interface might not be possible, because this must be part of the VideoFrameMetadata dictionary. Also is maplike even serializable?